source-control/git.md

```diff
@@ -211,8 +211,76 @@ git stash
 git stash pop
 ```
 
-Or you can simply move the current state into a new branch:
+Or you can move the current state into a new branch:
 
 ```bash
 git stash branch <new_branch_to_save_changes>
 ```
+
+### Working with images, video and other binary content
+
+Avoid committing frequently changed binary files, such as large images, video or compiled code to your git repository. Binary content is not diffed like text content, so cloning or pulling from the repository may pull each revision of the binary file.
+
+One solution to this problem is `Git LFS (Git Large File Storage)` - an open source Git extension for versioning large files. `Git LFS` replaces the binary files with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or Azure DevOps.
```

Comment:
Nit: One thing we may want to include here is a note about how Git LFS influences security boundaries of code/data in GitOps. Since files are not stored in the repository, it's worth clarifying how that might affect customers who are particularly sensitive to issues like source code leak and developer asset theft.

Linking to something like this would be helpful for people dealing with those more data-boundary-aware customers: https://github.com/git-lfs/git-lfs/blob/master/docs/api/authentication.md

(This is especially important given that you note a couple sections down that Azure Repos don't support SSH connections for Git LFS)

Reply:
Fixed